cli/build.gradle

```diff
@@ -88,6 +90,8 @@ dependencies {
 	compile group: "org.eclipse.aether", name: "aether-transport-wagon", version: "1.0.2.v20150114"
 	compile group: "org.eclipse.aether", name: "aether-util", version: "1.0.2.v20150114"
 	compile group: "org.fusesource.jansi", name: "jansi", version: "1.17.1"
+	compile group: "org.glassfish.jersey.core", name: "jersey-client", version: "2.28"
```

Comment:
Do we need to pull anything from these dependencies into our jar?  I don't see anything in the -privatepackage for these.